exercises/pangram/src/test/java/PangramsTest.java

```diff
@@ -66,4 +66,10 @@ public void pangramWithNonAsciiCharacters() {
     public void panagramInAlphabetOtherThanAscii() {
         assertFalse(Pangrams.isPangram("Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства."));
     }
+
+    @Ignore
+    @Test
+    public void upperAndLowerCaseVersionsOfTheSameCharacterShouldNotBeCountedSeparatelyIsNotPangram() {
```

Comment:
Sorry to be very picky and possibly confusing, but I don't think this one needs `IsNotPangram` at the end. The other tests are specifying that some kinds of phrases are or aren't recognized as pangrams, e.g. pangram in another alphabet is not recognized as a pangram whereas an all lowercase pangram is. However, this test is checking for a common error in the implementation, i.e. counting upper and lowercase letters as different letters. Therefore I think `upperAndLowerCaseVersionsOfTheSameCharacterShouldNotBeCountedSeparately` is an okay name :) I hope that makes sense! :)